index.html

```diff
@@ -93,11 +93,27 @@ <h1 class="title3">Gifts for all Occasions</h1>
                 <div style="font-size: 32px; margin-bottom: 23px;">Our Best Blogs Ever</div>
                 <div style="font-size: 18px;">"Try our blog to find the best tips and tricks to select your gift</div>
             </div>
-            <div>
-                <div>
-
+            <div class="blog-card-feature">
+                <div class="stn6-bg"></div>
```

Comment:
the background image should wrap content, should not define an empty tag `stn6-bg`